src/components/AssignTeamPopover/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -58,6 +58,8 @@ Object {
   "jurisdictionId": "outpost-number-one",
   "name": "plan-assignment-form-outpost-number-one",
   "organizations": Array [],
+  "parentAssignments": undefined,
+  "parentIds": undefined,
```

Comment:
why is this undefined?

Reply:
We have those parameters as optional so if not passed they will be undefined. Have added tests for scenarios were the parameters are passed.